frontend/src/views/Org/SshCaPage/components/SshCertificateTemplateModal.tsx

```diff
@@ -0,0 +1,349 @@
+import { useEffect } from "react";
```

Comment:
I think admins should be able to configure the extensions right? 
<img width="333" alt="image" src="https://github.com/user-attachments/assets/8636a890-0cef-4165-ad33-498a7cf76ba8">

I envision it to be something like the following
<img width="597" alt="image" src="https://github.com/user-attachments/assets/bef3424d-77eb-4319-b541-0ea0bb3cb870">

Reply:
Note: Will leave extension configurability to Infisical SSH V2 / later improvement.